src/entity/mesh.hpp

```diff
@@ -213,8 +213,9 @@ generate_mesh(T voxel_object) {
             unsigned int width = 0, height = 0;
 
             counter = 0;
-            for (unsigned int j = 0; j < object_geometry[dims_index_2]; ++j)
-                for (unsigned int i = 0; i < object_geometry[dims_index_1];) {
+            for (unsigned int j = 0; j < object_geometry[dims_index_2]; ++j){
+                unsigned int i = 0;
+                while (i < object_geometry[dims_index_1]) {
```

Comment:
```suggestion
            for (size_t j = 0; j < object_geometry[dims_index_2]; ++j){
                size_t i = 0;
                while (i < object_geometry[dims_index_1]) {
```

Reply:
I think I'm going to refactor this function in the future

Reply:
#50